.circleci/config.yml

```diff
@@ -322,7 +322,7 @@ jobs:
             
             shopt -s globstar
             
-            SPECS=$(circleci tests glob "e2e_tests/features/**/*.feature" | circleci tests split --split-by=timings --show-counts)
+            SPECS=$(circleci tests glob "e2e_tests/features/e2eLocalTests.feature" | circleci tests split --split-by=timings --show-counts)
```

Comment:
I think this might need to be e2e_tests/features/e2eLocalTests/*.feature

And similar for the one above